DISCORD.md

```diff
@@ -0,0 +1,4 @@
+# Official PojavLauncher Discord server
+
+the only official invite for the PojavLauncher server, beware of invites created by other users.
```

Comment:
Fix the grammar error

Reply:
i seriously don't see anything wrong in it

Reply:
The "the" at "the only" should have a capitalized T?

Reply:
Aighty, done